.github/workflows/turing.yaml

```diff
@@ -389,6 +389,7 @@ jobs:
           GOPATH: ${{ github.workspace }}/api/.go
           TEST_ID: ${{ github.run_id }}
           MOCKSERVER_ENDPOINT: http://mockserver
```

Comment:
Nit: Shall we call this `MOCKSERVER_HTTP_ENDPOINT` for consistency? Similar comment about the change in `api/e2e/test/00_all_e2e_test.go`.